src/ert/scheduler/scheduler.py

```diff
@@ -190,13 +190,17 @@ async def _publisher(self) -> None:
             ping_interval=60,
             close_timeout=60,
         ):
-            while True:
-                event = await self._events.get()
-                if event == CLOSE_PUBLISHER_SENTINEL:
-                    self._publisher_done.set()
-                    return
-                await conn.send(event)
-                self._events.task_done()
+            try:
+                while True:
+                    event = await self._events.get()
+                    if event == CLOSE_PUBLISHER_SENTINEL:
+                        self._publisher_done.set()
+                        return
+                    await conn.send(event)
+                    self._events.task_done()
+            except ConnectionClosed:
+                logger.debug("Connection to EnsembleEvalutor went down, reconnecting.")
```

Comment:
Hmm, the exception will happen in `conn.send`. The content of `event` will then probably be lost.